### ISSUE TITLE
AWS Image running EOL version of Ubuntu

### ISSUE THREAD

**Issue by joshuacronemeyer** [RedashCommunity/redash#8]
### Issue Summary

I'm running redash in AWS from the published images. Today I got an alert from AWS that the version of Ubuntu we're using is EOL. https://ubuntu.com/blog/18-04-end-of-standard-support 

Hopefully this is the appropriate place to raise this issue. 

### Technical details:

* Redash Version: AMI Docs say they are version 8
* Browser/OS: Welcome to Ubuntu 18.04.6 LTS (GNU/Linux 5.4.0-1069-aws x86_64)
* Followed this guide https://redash.io/help/open-source/setup

**Comment by justinclift** [comment]
Thanks, yep this is the right place to raise it.  One of our developer members pointed out the same thing yesterday as well. :smile:

Our plan is to create new images in the near future (next few days, or next week maybe?) with outstanding security patches applied.

Hopefully updating to a newer Ubuntu base can be done at the same time, but depending on the work required (unsure yet) it might need to come in a subsequent minor release.

**Comment by joshuacronemeyer** [comment]
Ok thanks. I volunteer to test a new one when you guys have that done. 👍 

Do you know if the new AMI will be version 10? (I'll probably be needing a guide for handling the migration from the old image to the new one.)

**Comment by justinclift** [comment]
Yeah, we're looking to just do a very minor revision bump of the existing version 10, fixing some security problems that have come to light since the last release.

No database migrations should be needed, unless something weird turns up while we're putting it together.

**Comment by justinclift** [comment]
The Docker Hub location is here, if that's useful info:

https://hub.docker.com/r/redashcommunity/redash/

We'll let people know when it's ready to try out via this GitHub repo, and on our new [website-which-is-just-currently-a-placeholder](https://redash.community).
